ansible/roles/dashmate/tasks/main.yml

```diff
@@ -173,8 +173,53 @@
   changed_when: true
   when: dashmate_branch is defined
 
-# TODO: we should use status command to figure out if dashmate is running
-#   to get the status from the response
+- name: Get status
+  ansible.builtin.command: "{{ dashmate_cmd }} status --format=json"
+  become: true
+  become_user: dashmate
+  changed_when: true
+  args:
+    chdir: '{{ dashmate_cwd }}'
+  register: dashmate_status
+
+- name: Get service status
+  ansible.builtin.set_fact:
+    core_service_status: "{{ dashmate_status.stdout | from_json | json_query('core.serviceStatus') }}"
+    tenderdash_service_status: "{{ dashmate_status.stdout | from_json | json_query('platform.tenderdash.serviceStatus') }}"
```

Comment:
It seems we need to get all platform service statuses otherwise it will be inconsistency and conflict with the check in dashmate https://github.com/dashpay/platform/blob/2e984df0302fa94bdf187bd893328ec58f390aa0/packages/dashmate/src/docker/DockerCompose.js#L75

We using this function in start and restart commands I believe